app/controllers/problem_reports_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+class ProblemReportsController < ApplicationController
+  def index
+    @problem_reports = ProblemReport.order(id: :desc)
+    @filter = filter_message
+    if @filter == 'Unfixed and Critical'
+      @problem_reports = @problem_reports.where(is_critical: true,
+                                                is_fixed: false)
+    elsif @filter == 'Unfixed and Fixable'
+      @problem_reports = @problem_reports.where(is_fixable: true,
+                                                is_fixed: false)
+    end
+  end
+
+  def new; end
+
+  def create
+    if create_new_report.save
+      redirect_to problem_reports_path, notice: 'Success'
+    else
+      redirect_to new_problem_report_path, notice: 'Fail'
+    end
+  end
+
+  def update
+    update_remarks
+    %w[is_fixable is_fixed is_blocked is_critical].each do |a|
+      if params[a]
+        val = !@report.public_send(a)
+        @report.public_send("#{a}=", val)
+      end
+    end
+
+    @report.save
+    redirect_to problem_reports_path
+  end
+
+  def update_remarks
+    @report = ProblemReport.find(params[:id])
+    @report.last_update_user_id = current_user.id
+    @report.remarks = params[:remarks] if params[:remarks]
```

Comment:
Lol you're not saving your report. Anyways, `@report.update(last_update_user: current_user, ...)` will do.